.github/workflows/release.yml

```diff
@@ -5,8 +5,12 @@ on:
     tags:
       - '**'
 
+permissions: {}
 jobs:
   build:
+    permissions:
+      contents: write # to push pages branch (peaceiris/actions-gh-pages)
```

Comment:
confirmed that this is enough to make a release via `hub release create`.

https://stackoverflow.com/a/69941765/814928